pkg/kubecost/allocation.go

```diff
@@ -844,6 +895,12 @@ func (a *Allocation) add(that *Allocation) {
 		a.ProportionalAssetResourceCosts.Add(that.ProportionalAssetResourceCosts)
 	}
 
+	// If both Allocations have SharedCostBreakdowns, then
+	// add those from the given Allocation into the receiver.
+	if a.SharedCostBreakdown != nil && that.SharedCostBreakdown != nil {
+		a.SharedCostBreakdown.Add(that.SharedCostBreakdown)
+	}
```

Comment:
What if just one of the Allocations has a SCB? Shouldn't it still be added to the result?